www/js/diary/services.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 import angular from 'angular';
-import { getFormattedTimeRange, motionTypeOf } from './diaryHelper';
+import { getBaseModeByKey, getBaseModeOfLabeledTrip } from './diaryHelper';
```

Comment:
nit: presumably you don't need this any more?

Reply:
Yeah, none of these external functions are needed anymore

There are only a few functions left in this file though - basically just `readAllCompositeTrips` and `readUnprocessedTrips` and the smaller functions they depend on.

I think we can incorporate those into `timelineHelper.ts` on the next pass and `services.js` will no longer be needed